io.mpv.Mpv.yml

```diff
@@ -614,8 +614,17 @@ modules:
           - mkdir -p /app/share/appdata
           - mv $FLATPAK_ID.appdata.xml /app/share/appdata/$FLATPAK_ID.appdata.xml
           - chmod 644 /app/share/appdata/$FLATPAK_ID.appdata.xml
+          - sed -i "s/Exec=mpv/Exec=mpv-wrapper/" etc/mpv.desktop
 
 # Scripts for mpv
+  - name: mpv-wrapper
+    buildsystem: simple
+    build-commands:
+      - install -D mpv-wrapper /app/bin/mpv-wrapper
```

Comment:
I propose to install it as `mpv` and rename mpv binary to `mpv-bin` instead. This way you wouldn't need change flatpak commands everywhere.

Reply:
For the record, this is also important because I have a "generate/manage tab-completion-friendly command-line launchers" [script](https://gist.github.com/ssokolow/db565fd8a82d6002baada946adb81f68) and one of the stack of transformations it has to do when harvesting the `command` name is removing  `-wrapper` and `_wrapper` suffixes.

For better or for worse, to make Flatpak packages an acceptable replacement for how I use distro packages, I've Hyrum's Law'd the libflatpak API equivalent to `flatpak info -m io.mpv.Mpv | grep 'command='`.